cmd/coder/sync.go

```diff
@@ -99,7 +98,7 @@ func (cmd *syncCmd) Run(fl *pflag.FlagSet) {
 	if rsyncErr != nil {
 		flog.Info("Unable to determine remote rsync version.  Proceeding cautiously.")
 	} else if localVersion != remoteVersion {
-		flog.Fatal(fmt.Sprintf("rsync protocol mismatch. %s.", localVersion, rsyncErr))
+		flog.Fatal("rsync protocol mismatch: local = %v, remote = %v", localVersion, rsyncErr)
```

Comment:
Why is this in this PR? just curious seems like it's own thing

Reply:
Was causing some linting errors so I figured I'd clean it up.

Reply:
Won't happen again once we get proper CI.